src/main/java/frc/robot/subsystems/intake/IntakeConstants.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot.subsystems.intake;
+
+import static edu.wpi.first.units.Units.Radians;
+
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.MutableMeasure;
+
+public class IntakeConstants {
+    public static final double ANGLE_RATIO = 0;
+    public static final MutableMeasure<Angle> DOWN_ANGLE = MutableMeasure.zero(Radians);
+    public static final MutableMeasure<Angle> UP_ANGLE = MutableMeasure.zero(Radians);
+
+    public enum IntakePose {
+        UP(IntakeConstants.UP_ANGLE),
+        DOWN(IntakeConstants.DOWN_ANGLE);
+        MutableMeasure<Angle> intakePose;
+
+        private IntakePose(MutableMeasure<Angle> intakePose) {
```

Comment:
No need for private keyword here.

Reply:
OK puppy

Reply:
381418fb60abfd384af6a39eaa907c7483a6ecf4